### PR TITLE
Tests on fastapi backend

### PR DESCRIPTION
**Pull request by Zaker237**
This PR provide the tests for the Backend endpoints.

why: before deploying the project we need the make sure that we don't get any unexpected mistakes.

### CODE REVIEW

File: fastapi/onacut/routers/districts.py [hunk 1]
```diff
@@ -43,7 +44,16 @@ def get_district(district_id: int, db: Session = Depends(get_db)):
     responses={403: {"description": "Operation forbidden"}},
 )
 def create_district(district: DistrictCreateSchema, db: Session = Depends(get_db)):
-    return {}
+    # check if the provided city's id exists in the database
+    city = db.query(CityModel).filter_by(id=district.city_id).first()
+    if not city:
+        raise HTTPException(status_code=400, detail="Bad city's id!")
```
[the remark this answers is not in the document]
Comment (Sanix-Darker) [reply fastapi/onacut/routers/districts.py:50]: working on it, if you agree it should be done !

File: fastapi/onacut/tests/test_alerts.py [hunk 2]
```diff
@@ -2,5 +2,92 @@
...
+        "type": "electricity",
+        "date": "2022-10-30",
+        "begin_time": "18:00:00",
+        "region_id": 4,
+        "longitude": 11.124576,
+        "lattitude": 4.012475,
+        "city_id": 1,
+        "district_id": 1,
+        "end_time": "20:30:00",
+    }
```
[the remark this answers is not in the document]
Comment (Sanix-Darker) [reply fastapi/onacut/tests/test_alerts.py:21]: working on it, if you agree it should be done !

File: fastapi/onacut/routers/cities.py [hunk 3]
```diff
@@ -43,7 +44,16 @@ def get_city(city_id: int, db: Session = Depends(get_db)):
...
-    return {}
+    # check if the provided region's id exists in the database
+    region = db.query(RegionModel).filter_by(id=city.region_id).first()
+    if not region:
+        raise HTTPException(status_code=400, detail="Bad region's id!")
+
+    db_city = CityModel(**city.dict())
+    db.add(db_city)
+    db.commit()
+    db.refresh(db_city)
```
[the remark this answers is not in the document]
Comment (Zaker237) [reply fastapi/onacut/routers/cities.py:55]: yes
with the refresh

File: fastapi/onacut/tests/test_alerts.py [hunk 4]
```diff
@@ -2,5 +2,92 @@
...
+        "date": "2022-10-30",
+        "begin_time": "18:00:00",
+        "region_id": 4,
+        "longitude": 11.124576,
+        "lattitude": 4.012475,
+        "city_id": 2,
+        "district_id": 0,
+        "end_time": "18:00:00",
+    }
+
```
[the remark this answers is not in the document]
Comment (Zaker237) [reply fastapi/onacut/tests/test_alerts.py:82]: @Sanix-Darker just go on!!!!

**Review by Sanix-Darker — APPROVED** [review]
LGTM !

+ (in fact, just to merge, so that we could process #151 )

**Pull request merged**